src/zenml/integrations/langchain/__init__.py

```diff
@@ -0,0 +1,52 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Initialization of the langchain integration."""
+
+import sys
+from typing import List, Optional
+from zenml.integrations.constants import LANGCHAIN
+from zenml.integrations.integration import Integration
+
+from zenml.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+class LangchainIntegration(Integration):
+    """Definition of langchain integration for ZenML."""
+
+    NAME = LANGCHAIN
+    REQUIREMENTS = []
+
+    @classmethod
+    def activate(cls) -> None:
+        """Activates the integration."""
+        if sys.version_info < (3, 8):
```

Comment:
There are multiple checks of this, for example here and in the `integrations/__init__`, can we get rid of some of them?